spring-cloud-dataflow-server-local/src/main/resources/META-INF/kafka-apps.properties

```diff
@@ -37,3 +37,8 @@ sink.tcp=maven://org.springframework.cloud.stream.app:tcp-sink-kafka:@registry.a
 sink.throughput=maven://org.springframework.cloud.stream.app:throughput-sink-kafka:@registry.app.defaultVersion@
 sink.websocket=maven://org.springframework.cloud.stream.app:websocket-sink-kafka:@registry.app.defaultVersion@
 task.timestamp=maven://org.springframework.cloud.task.app:timestamp-task:@registry.app.defaultVersion@
```

Comment:
@cppwfs I am not sure I understand the above. I think it is just a convenient we just register them via explicit location (task-apps.properties in this case). Irrespective of the location, these apps are always registered as `task` apps.

@sabbyanandan I thought the reverse. By not keeping `task` applications in a separate file, we add extra step to add the tasks in all the binder properties files (and duplicating them).